CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to CCMMF
+
+<!-- 
+TODO:
+- Add preferred languages
+- Code style
+- What contributions go to PEcAn or SIPNET, what to CCMMF
+  - Types of output: workflow, documentation, sipnet, monitoring
+- Guidance for tests, documentation, NEWS
+- Authorship, CITATION.cff
+- what goes in private vs public discussions?
+-->
+
+The California Cropland Monitoring and Modeling Framework (CCMMF) is an integrated pipeline
+built on the [SIPNET biogeochemistry model](https://github.com/pecanproject/sipnet) and 
+[PEcAn](https://github.com/pecanproject/) modeling, informatics, and statistical inference workflow 
+engine.
+
+This is a guide for contributors and users of the software.
+
+## How to contribute
+
+We welcome contributions from the community to help improve and apply CCMMF including documentation, 
+software, and data products. 
+
+### Communication
+
+Feel free to reach out directly with ideas, questions, and suggestions through 
+our [GitHub Discussions](https://github.com/orgs/ccmmf/discussions) and join the CCMMF channel 
```

Comment:
the CCMMF channel is currently private -- should we change that?

Reply:
Perhaps we should create a new public channel for this purpose?

Reply:
```suggestion
our [GitHub Discussions](https://github.com/orgs/ccmmf/discussions) and join the #ccmmf channel 
```